app/lib/three_scale/swagger/specification.rb

```diff
@@ -145,12 +155,18 @@ def swagger_2_0?
         @doc.fetch("swagger", 0).to_f >= 2.0
       end
 
+      def openapi_3_0?
+        @doc.fetch("openapi", 0).to_f >= 3.0
```

Comment:
I think there is no need to use `fetch` in this case

```suggestion
        @doc['openapi'].to_f >= 3.0
```